### PR TITLE
Get support for LAG Type

### PR DESCRIPTION
**Pull request by ArthiSivanantham**

### CODE REVIEW

**Comment by ArthiSivanantham** [comment]
show_po
show_po_id

File: src/CLI/renderer/templates/show_interface_portchannel.j2 [hunk 1]
```diff
@@ -1,12 +1,53 @@
...
+{% set minlinks = po_conf_intf['min_links'] %}
+{% endif %}
+
+
+{% if po_conf_intf['name'] == po_intf['lagname'] and 'fallback' in po_conf_intf %}
+{% set fallback = po_conf_intf['fallback']  %}
+{% endif %}
+
+{{po_intf['lagname']}} is {{po_intf['admin_status']}}, line protocol is {{po_intf['oper_status']}}
+mode {{mode}} min-links {{minlinks}} fallback {{fallback}}
```
Comment (Tejaswi-Goel) [inline src/CLI/renderer/templates/show_interface_portchannel.j2]: can we please make all attributes comma separated or in new line.

**Comment by ArthiSivanantham** [comment]
show_po_id
show_po

File: src/CLI/renderer/templates/show_interface_portchannel.j2 (hunk 1 above)
Comment (ArthiSivanantham) [reply src/CLI/renderer/templates/show_interface_portchannel.j2]: Modified display strings

**Review by bhavini-gada — APPROVED** [review]

File: src/translib/transformer/sw_portchannel.go [hunk 2]
```diff
@@ -167,7 +173,7 @@ func can_configure_fallback(inParams XfmrParams) error {
            var mode string
            e = get_lag_type(inParams.d, &ifKey, &mode)
            if e == nil {
-               errStr := "Fallback option cannot be re-configured for an already existing PortChannel: " + ifKey
+               errStr := "Fallback option cannot be configured for an already existing PortChannel: " + ifKey
```
Comment (joyas-joseph) [inline src/translib/transformer/sw_portchannel.go:176]: Setting the same mode should be a no-op. So, why have we changed "re-configured" to "configured"

Comment (bhavini-gada) [reply src/translib/transformer/sw_portchannel.go:176]: This also gets called for portchannel that is already configured, not necessarily for updating the same mode, no-op case may need additional check.

**Pull request merged**